pkg/pool-weighted/test/ManagedPool.test.ts

```diff
@@ -319,7 +327,17 @@ describe('ManagedPool', function () {
 
     context('with the max swap fee', () => {
       sharedBeforeEach('set swap fee to the max value (< 100%)', async () => {
-        await pool.setSwapFeePercentage(owner, MAX_SWAP_FEE_PERCENTAGE);
+        // In practice, a separate contract would call `updateSwapFeeGradually` using `block.timestamp` both as start
+        // and endTime to make the change immediately.
+        const blockTimestamp = (await currentTimestamp()).add(1);
+        await setNextBlockTimestamp(blockTimestamp);
```

Comment:
```suggestion
        const blockTimestamp = await currentTimestamp();
        await setNextBlockTimestamp(blockTimestamp.add(1));
```

Reply:
I've rephrased this one a bit.

If we send `start == end` to `updateSwapFeeGradually` and the timestamp is in the past, the function will resolve start time to the present, and revert because `resolvedStart > end`. For this to work 'instantly', we need to mock the call to `updateSwapFeeGradually` with `block.timestamp`.